### DISCUSSION TITLE
device action getting slowly

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#13257] [General]
Hi,

I'm not sure wha happens. I run zigbee2mqtt with 156 devices. 
The following phenomenon has been occurring for the last 3 days:
When I activate an action in the frontend (e.g. switch on a bulb), it sometimes takes longer for the action to be carried out on the device.
I'm using the latest dev version on a Raspi4 and the Slaesh's Stick.
After about 30 minutes it comes to a standstill and actions are no longer forwarded to the devices.
After a restart of zigbee2mqtt, it runs smoothly again for about 10 minutes before the phenomenon occurs again.
Maybe someone has the same experience and possibly a solution or a tip?
The log level is set to "Error".

**Comment by Koenkk** [comment]
Can you provide the herdsman debug logging of this?

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

**Reply by Burki24** [reply]
> Can you provide the herdsman debug logging of this?
> 
> See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

I did an  DEBUG=zigbee-herdsman* npm start>debug_20220723_1821.txt (hope that was the right way)
[debug_20220723_1821.zip](https://github.com/Koenkk/zigbee2mqtt/files/9174159/debug_20220723_1821.zip)

Thanks in advance for your help.

**Comment by Burki24** [comment]
I made a second dump. Same situation, it starts running well. After more than one hour it starts to ignore send payloads.
[debug_20220723_2028.zip](https://github.com/Koenkk/zigbee2mqtt/files/9174533/debug_20220723_2028.zip)

**Reply by Koenkk** [reply]
Can you try to power down `Küche/Klima/Luftqualität`, `Aussen/Strom/Gesamt/Verbrauch` and `Büro/Sicherheit/RadarSensor` and see if the issue still occurs?

**Reply by Burki24** [reply]
I tried, but no positive result. So now I install all new, hoping that it will run smoothy again. I also changed the stick from Slaesh's tu Sonoff. But cannot start zigbe, getting error "network commissioning timed out." I read the fails to start manual but found no solution. So I will try tomorrow again.

Thanks for your support. If it will not run tomorrow, I'm sure, I#m coming back.

Greetings
Burkhard

**Comment by Burki24** [comment]
Hello @Koenkk ,

the system is running fine again. But I can't switch to dev. When I will switch to dev, I get the error: `error: pathspec 'dev' did not match any file(s) known to git`

When I try to look at the existing brnaches, I get following:
```
pi@Z2M-Server:/opt/zigbee2mqtt $ git branch -a
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/master
```
Do you know, why or what I'm doing wrong?

Thanks and greetz
Burkhard

UPDATE: Could solve the problem. after using `git clone --depth 1 https://github.com/Koenkk/zigbee2mqtt.git /opt/zigbee2mqtt` without `--depth 1` it shows me the dev branch.

So no more help needed. Thanks.
